client/deposits/overview/index.js

```diff
@@ -196,7 +196,7 @@ const DepositsOverview = () => {
 								) }
 								value={ getAmount(
 									overview.last_deposit,
-									overview.account.default_currency
+									wcpaySettings.currencies.default
```

Comment:
@vbelolapotkov @jrodger : this is the change I mentioned at the team chat today (along with server-side change, which drops the outdated property injection).

Reply:
So am I right in thinking that `overview.account.default_currency` isn't currently used in a released version of the plugin?

So merging this before the release has the advantage that we don't need to support `default_currency`, we can drop it from the server almost immediately (we only have to worry about breaking development environments).

The downside is "rushing" this PR to get it merged when supporting `default_currency` for a while might not be a big deal?

Reply:
Yes, yes, and yes =)

Reply:
Speaking of this, I haven't reviewed the whole PR, but I have higher other concerns about this approach: 
1. IMO it's fine to return currency value with deposits overview object.
2. We should avoid using global variables in components in JS. I'd say consuming it either via the hook or single method is better approach than using directly.

What do you think?

Reply:
No rush, see new comments below.

Reply:
Reverted in 1d195aad823c4652a5cbd8d221a4ff60444d6585